src/main/resources/application-datasource.yml

```
@@ -0,0 +1,36 @@
+    driver-class-name: org.h2.Driver
+    url: jdbc:h2:mem:testdb;MODE=MySQL;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password:
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
+    show-sql: true
+  h2:
+    console:
+      enabled: true
+      path: /h2
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)